server/views/components/goal-summary-card/_goalSummaryCardContent-goalInStatus-COMPLETED.njk

```diff
@@ -1,19 +1,30 @@
 <div class="govuk-summary-card__content">
   {% include './_goalSteps.njk' %}
 
-  {% if params.goal.notesByType.GOAL.length > 0 %}
-    {% set goalNote = params.goal.notesByType.GOAL[0] %}
-    <details class="govuk-details app-notes-expander" data-qa="overview-notes-expander">
-      <summary class="govuk-details__summary">
-            <span class="govuk-details__summary-text">
+  {% if params.goal.status === 'COMPLETED' %}
+      {% if params.goal.notesByType.GOAL_COMPLETION.length > 0 %}
+        {% set goalCompletionNote = params.goal.notesByType.GOAL_COMPLETION[0] %}
+        <p class="govuk-heading-s">Goal completion note</p>
```

Comment:
I think I need to do a similar thing for the archived goals template (TBC this morning in 3As with Jess), but on the assumption I do I was thinking that semantically this is a heading, especially as you are using a heading classname on it.
I was thinking of a H3, but I've not looked at the rest of the page/component hierarchy yet to see if h3 is structurally and semantically right (it might be better as a h4 for example?)

What do you think?

Reply:
Makes sense, changed to h3.